apps/docs/src/guide/types/date-time.md

```diff
@@ -0,0 +1,60 @@
+# Date Time
+
+To allow for easier manipulation of date and time, the SDK exports the `DateTime` class, which is a wrapper around the `Date` class ([reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)). Below we will go over the methods of instantiation, utility functions, and time formats.
+
+Why is this useful?
+Internally the transactions and other time/date assets are encoded using the **TAI64** format. We return a `IDateTime` interface, to allow of easier conversion and formatting between the two formats.
```

Comment:
Since here is the first mention of 'tai64' on this page, can you make the first occurrence of the term a link to the section where you have explained the format down below on the same page?

Reply:
That seems wise!